setup.py

```diff
@@ -14,68 +14,54 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""simpl packaging and installation."""
+"""simpl packaging, installation, and package attributes."""
 
-import ast
-import re
+import os
 from setuptools import find_packages
 from setuptools import setup
 
 
-DEPENDENCIES = [
+src_dir = os.path.dirname(os.path.realpath(__file__))
+
+about = {}
+with open(os.path.join(src_dir, 'simpl', '__about__.py')) as abt:
+    exec(abt.read(), about)
```

Comment:
@Pablosan  Can you elaborate on why you prefer exec over eval?

I'm not sure eval is an option here. They both pretty much compile a string to code and execute it, so I might be missing something?